happi/cli.py

```diff
@@ -297,45 +295,45 @@ def edit(ctx, name: str, edits: List[str]):
     client = ctx.obj
 
     logger.debug('Starting edit block')
-    item = client.find_item(name=name)
+    item = None
+    try:
+        item = client.find_item(name=name)
+    except SearchError as e:
+        raise click.ClickException(str(e))
```

Comment:
Something a bit more descriptive perhaps?
```suggestion
        raise click.ClickException(f"Could not find item {name!r}: {e}")
```